src/main/services/keycloakAuthProvider.ts

```
@@ -38,6 +38,7 @@ export class KeycloakAuthProvider implements BaseAuthProvider {
             deauthenticated(req);
         } catch (error) {
             console.error('Logout error:', error);
```

ESLint

[error] Unexpected console statement.

(no-console)